source/_docs/solr-drupal-8.md

```diff
@@ -100,6 +100,15 @@ The version of Solr on Pantheon is Apache Solr v3.6. To accommodate this older v
 The following code changes are required before Solr can be safely uninstalled and disabled:
 {% include("content/remove-addons/d8-solr.html")%}
 
+## Troubleshooting
+
+### Solr Verification Check
+Because we are posting the 4.x schema to a 3.x solr instance, the schema verification check can fail and prevent indexing. You can disable the schema check by adding the following line to the `public function defaultConfiguration` block:
```

Comment:
So as the module exists now, code changes aren't required to fix this.  It might be worth pulling from the screenshots I attached here: https://github.com/pantheon-systems/documentation/issues/4246#issuecomment-469061878

Or some other method of communication that checking the box that says `Skip schema verification` in the UI is the only thing required to get it working.

The code changes I made in the other PR will make it skip schema verification permanently for all sites rather than requiring devs setting up Solr to remember and check the box during the setup phase.